### ISSUE TITLE
progress icon has the infamous triangle

### ISSUE THREAD

**Issue by water111** [open-goal/jak-project#2183]

**Comment by water111** [comment]
I started looking at this. I'm not sure entirely what the progress icon is, so I'm looking at the triangle during the intro cutscene and hoping it's the same problem.

I found that this code was commented out in `drawable.gc`:
```
      (if (-> *texture-pool* update-flag)
          (update-warp-and-hud *texture-pool*)
          )
```

Adding this caused different incorrect textures. The second overlay texture in the intro is a checkerboard, which is the placeholder.
```
[53:03:089] [warn] Failed to find texture at 448, using random
```
indicates that the PC renderer thinks `tbp = 448` points to unknown data. Either it is not initialized, or it is pointing to the middle of texture data.

I see that the `update-warp-and-hud` relocates this texture at 448 (448 * 64 = 28672). This doesn't actually do an upload, just marks the texture's eventual destination as 448.
```
#<texture-page village1-minimap :length 12 :dest #x0 :size 422K @ #x19dae54> to 28672
```

So possibly:
- this upload never happens because the texture system thinks this texture isn't drawn.
- the upload happens, but the C++ misses it
- the upload happens, but something writes over it after

**Comment by water111** [comment]
![Image](https://user-images.githubusercontent.com/48171810/215918941-94ce2e69-bb3e-4d89-b9d9-f5ce687d08e0.png)

**Comment by water111** [comment]
fixed in #2184
